go.mod

```diff
@@ -1,18 +1,17 @@
 module github.com/ory/oathkeeper-maester
 
-go 1.12
+go 1.15
```

Comment:
Since you changed the required go version, please update it also in the [CI config](https://github.com/ory/oathkeeper-maester/blob/master/.circleci/config.yml#L5) otherwise, it won't be able to run

Reply:
Thanks for the advice ❤️ 
I will update kind, too